raster/src/main/scala/geotrellis/raster/op/focal/Kernel.scala

```diff
@@ -25,9 +9,15 @@ import geotrellis.vector.Extent
  * Represents a neighborhood that is represented by
  * a tile.
  */
-case class Kernel(tile: Tile) {
+case class Kernel(tile: Tile) extends Neighborhood {
   if(tile.rows != tile.cols) sys.error("Kernel tile must be square")
```

Comment:
Should this be `require`? It produces `IllegalArgumentException`, subtype of `RuntimeException` produced here.